src/views/Settings.vue

```diff
@@ -104,6 +119,22 @@ export default {
           this.messages = response.data;
         });
     },
+    updateActiveSettings() {
+      this.activeSettings.messageId = this.activeSettings.message.id;
+      ApiService.updateActiveSettings(this.activeSettings, this.activeSettings.id);
+    },
+    showActiveSettings() {
+      this.showMessages = !this.showMessages;
+      this.messagesBtnText = this.showMessages ? "Hide all messages" : "Show all messages";
+    },
+    isMessageExisting() {
```

Comment:
I suggest loading **lodash** lib to project. There are nice methods that can help with filter (and a lot other methods) and with using less code.
Example for this function
```
isMessageExisting() {
   return _.some(this.messages, { 'text': this.activeMessage });
}
```